### ISSUE TITLE
TypeError: __init__() got an unexpected keyword argument 'html'

### ISSUE THREAD

**Issue by Triquetra** [ruia-plugins/ruia-pyppeteer#10]
```
import aiohttp
import asyncio
from ruia_pyppeteer import PyppeteerRequest as Request


async def request_example():
    url = "https://deck.tk/07Pw8tfr"
    params = {
        'name': 'ruia',
    }
    headers = {
        'User-Agent': 'Python3.6',
    }
    async with aiohttp.ClientSession() as session:
        request = Request(url=url, method='GET', params=params, request_session = session , headers=headers, load_js=True)
        response = await request.fetch()
        html = await response.text()
        print(html)


if __name__ == '__main__':
    asyncio.get_event_loop().run_until_complete(request_example())
```
produces:

```
Traceback (most recent call last):                                                                                             
  File "C:\yourproject\spiders\yourspider.py", line 22, in <module>                            
    asyncio.get_event_loop().run_until_complete(request_example())                                                             
  File "c:\python39\lib\asyncio\base_events.py", line 642, in run_until_complete                                               
    return future.result()                                                                                                     
  File "C:\yourproject\spiders\yourspider.py", line 16, in request_example                     
    response = await request.fetch()                                                                                           
  File "C:\yourproject\env\lib\site-packages\ruia_pyppeteer\request.py", line 76, in fetch    
    response = PyppeteerResponse(                                                                                              
  File "C:\yourproject\env\lib\site-packages\ruia_pyppeteer\response.py", line 26, in __init__
    super(PyppeteerResponse, self).__init__(                                                                                   
TypeError: __init__() got an unexpected keyword argument 'html'                                                                
[I:pyppeteer.launcher] terminate chrome process... 
```

**Comment by howie6879** [comment]
Upgrade to v0.0.7:

```shell
pip install git+https://github.com/python-ruia/ruia-pyppeteer.git
```

**Comment by Triquetra** [comment]
Solved in v0.07
